src/test/data/tx_invalid.json

```diff
@@ -32,7 +32,7 @@
 ["Tests for CheckTransaction()"],
 ["No inputs"],
 [[["0000000000000000000000000000000000000000000000000000000000000100", 0, "HASH160 0x14 0x7a052c840ba73af26755de42cf01cc9e0a49fef0 EQUAL"]],
-"0100000000010000000000000000015100000000", "P2SH"],
+"01000000000000010000000000000000015100000000", "P2SH"],
```

Comment:
I think after your fixup commit this doesn't deserialize properly, which probably isn't what we're supposed to be testing here.